src/requestDeduplicator.ts

```diff
@@ -17,5 +17,6 @@ export default async function serve(id, action, args) {
     ongoingRequests.set(key, requestPromise);
   }
 
-  return ongoingRequests.get(key);
+  return;
+  ongoingRequests.get(key);
```

Comment:
this is weird

Reply:
omg 😆 . The lint fixed it in the wrong way